<?xml version="1.0"?>
<launch>
<arg name="laser_enabled" default="true" />

  <include file="$(find husky_gazebo)/launch/husky_empty_world.launch">
    <arg name="world_name" value="$(find husky_highlevel_controller)/worlds/singlePillar.world"/>
    <arg name="laser_enabled" value="$(arg laser_enabled)"/>
  </include>

  <node name="keyboard_control" pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" >
  </node>
  
  <!--
  <node name="husky_highlevel_controller" pkg="husky_highlevel_controller" type="husky_highlevel_controller" output="screen" >
	<rosparam command="load" file="$(find husky_highlevel_controller)/config/config.yaml" />
  </node>
  -->
  <node pkg="rviz" type="rviz" name="rviz" >
  </node>

</launch>
